api/environments/dynamodb/migrator.py

```diff
@@ -0,0 +1,64 @@
+from django.db.models import Prefetch
+from flag_engine.django_transform.document_builders import (
+    build_identity_document,
+)
+
+from environments.models import Environment
+from features.models import FeatureState
+from features.multivariate.models import MultivariateFeatureStateValue
+
+from .dynamodb_wrapper import DynamoIdentityWrapper
+from .types import DynamoProjectMetadata, ProjectIdentityMigrationStatus
+
+
+class IdentityMigrator:
+    def __init__(self, project_id):
+        self.project_metadata = DynamoProjectMetadata.get_or_new(project_id)
+
+    @property
+    def migration_status(self) -> ProjectIdentityMigrationStatus:
```

Comment:
It still feels to me like this should be a property on the `DynamoProjectMetadata` class.